src/console/ServiceInstallCommand.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace blink\console;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class ServiceInstallCommand
+ *
+ * @package blink\console
+ */
+class ServiceInstallCommand extends BaseService
+{
+    public $name = 'service:install';
+    public $description = 'Install {serviceName} as a system service';
+    public $bin = 'blink';
+
+    protected function configure()
+    {
+        $this->addOption('php', null, InputOption::VALUE_REQUIRED, 'Specify a custom php executable');
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->ensureRootPrivilege();
+        $this->ensurePkgConfigInstalled();
+
+        $unitFile = $this->getSystemUnitDir() . '/' . $this->serviceName . '.service';
+
+        file_put_contents($unitFile, $a = $this->getServiceConfig($input->getOption('php')));
```

Comment:
What if `$unitFile` was already existed?

In another word, what if another blink service having the same $serviceName?

Reply:
It will be overwritten, it would be better to have a confirm dialog.